microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -598,25 +542,112 @@ See also:
 * [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 131.
-* This checkbox is present in Microsoft Edge Stable 129.
+* This checkbox is present in Microsoft Edge Canary 133.
+* This checkbox is present in Microsoft Edge Stable 131.
 
 
-<!-- ------------------------------ -->
-#### Performance panel: enable server timings in the timeline
+<!-- ====================================================================== -->
+## Performance panel: enable server timings in the timeline
 <!-- checkbox has no "(?)" link pointing to this anchor wording -->
 
 _new as of Microsoft Edge Stable 129_
 
 Shows a **Server Timings** track in the **Main** section of the **Performance** tool, to display server timing events sent by your HTTP server. To learn more, see [Server timing](https://developer.mozilla.org/docs/Web/API/Performance_API/Server_timing) at MDN Web Docs.
-<!-- todo: desc -->
+<!-- todo old: desc -->
 
 See also:
 * [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 131.
-* This checkbox is present in Microsoft Edge Stable 129.
+* This checkbox is present in Microsoft Edge Canary 133.
+* This checkbox is present in Microsoft Edge Stable 131.
+
+
+<!-- ====================================================================== -->
+## Extension storage in Application panel
+<!-- checkbox has no "(?)" link pointing to this anchor wording -->
+
+_new as of Microsoft Edge Stable 131_
+
+<!-- todo: desc -->
+
+<!-- 
+See also: 
+* []()
+-->
+
+Status:
+* This checkbox is not present in Microsoft Edge Canary 133.
+* This checkbox is present in Microsoft Edge Stable 131.
+
+
+<!-- ====================================================================== -->
+## Floating entry points for the AI assistance panel
+<!-- checkbox has no "(?)" link pointing to this anchor wording -->
+
+_new as of Microsoft Edge Canary 133_
+
+<!-- todo: desc -->
+
+<!-- 
+See also: 
+* []()
+-->
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 133.
+* This checkbox is present in Microsoft Edge Stable 131.
+
+
+<!-- ====================================================================== -->
+## Performance panel: enable experimental performance insights
+
+_new as of Microsoft Edge Canary 133_
+
+<!-- todo: desc -->
```

Comment:
need desc: Performance panel: enable experimental performance insights

Reply:
The new Insights side panel can list multiple insights. Some insights might be experimental. This setting allows to display all insights, even the experimental ones.

Reply:
result:
Includes experimental insights in the **Insights** side panel in the **Performance** tool.